app/views/koi/settings/_form_fields.html.erb

```diff
@@ -7,7 +7,6 @@
     <%=
       assoc    = klass.reflect_on_association(attr)
       partial  = 'form_field'
-      partial += '_nested' if assoc && !assoc.belongs_to? && assoc.is_nested?
```

Comment:
This is the line of code that was crashing it. The association didn't have an `is_nested?` method.